packages/wdio-crossbrowsertesting-service/src/launcher.js

```diff
@@ -0,0 +1,35 @@
+var cbt = require('cbt_tunnels')
+
+export default class CrossBrowserTestingLauncher{
+    onPrepare(config) {
+        if(!config.cbtTunnel){
+            return
+        }
+        this.cbtOpts = Object.assign({
+            username: config.user,
+            authkey: config.key
+        }, config.cbtOpts)
+        this.tunnel = cbt
+        return new Promise(function(resolve, reject){
+            this.tunnel.start({'username': config.user,'authkey': config.key},function(err){
+                if(!err){  
+                    return resolve()
+                }else{
```

Comment:
I think there are some formatting issues here? I mean in whole file.

Reply:
I'm currently working to fix these issues and add unit test.